.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,27 @@
+name: Node.js CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
```

Comment:
> For some reason, the new guidance (at least from GitHub)

From the link, looks like that reason is:

> It is not recommended to cache node_modules, as it can break across Node versions and won't work with npm ci

I can't speak to the truth of that, but there you go. I'll add caching in a follow up PR.